modules/epomDspBidAdapter.js

```diff
@@ -0,0 +1,107 @@
+/**
+ * @name epomDspBidAdapter
+ * @version 1.0.0
+ * @description Adapter for Epom DSP and AdExchange
+ * @module modules/epomDspBidAdapter
+ */
+
+import { logError, logWarn, registerBidder } from '../src/adapters/bidderFactory.js';
+import { config } from '../src/config.js';
+const BIDDER_CODE = 'epomDsp';
+
+export const spec = {
+  code: BIDDER_CODE,
+
+  isBidRequestValid(bid) {
+    const globalSettings = config.getBidderConfig()[BIDDER_CODE]?.epomSettings || {};
+    const hasEndpoint = bid.params?.endpoint || globalSettings.endpoint;
+    return !!(hasEndpoint);
+  },
+
+  buildRequests(bidRequests, bidderRequest) {
+    const bidderConfig = config.getBidderConfig();
+    const globalSettings = bidderConfig['epomDsp']?.epomSettings || {};
+
+    const requests = bidRequests.map((bid) => {
+      const endpoint = bid.params?.endpoint || globalSettings.endpoint;
+      const payload = {
+        ...bid,
```

Comment:
fyi, without filtering this can be _a lot_ of data for some publisher auction configurations